client/src/components/ContactCitySection.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <div class='header-section'>
+      <div class='h2-header-large'>
+        {{ MESSAGES.CONTACT_YOUR_CITY_HEADER }}
+      </div>
+      <div>
+        {{ MESSAGES.CONTACT_YOUR_CITY_SUBHEADER }}
+      </div>
+    </div>
+    <div class='city-information'>
+      <div class='h2-header'> {{ this.city }}</div>
+      <a class='h2-header' :href='website' target='_blank'>{{ website }}</a>
+      <div class='h2-header'> Tel: {{ phoneNumber }}</div>
+    </div>
+  </div>
+</template>
+
+<script lang='ts'>
+
+import { City } from '../model/states/model/geo_data';
+import { defineComponent, PropType } from 'vue';
+import { ScorecardMessages } from '../assets/messages/scorecard_messages';
+
+interface CityInformation {
+  name: City;
+  website: string;
+  phoneNumber: string;
+}
+
+const CITY_INFO: CityInformation[] = [
```

Comment:
Did you consider using a map here instead of a list (e.g. `Map<City, CityInformation>`)? So we don't have to iterate through the list to look it up. 

(This is a really a nit because iterating through a list of 3 objects is basically free).

Reply:
I did, but have been hating javascript maps so was trying to avoid it (the initialization of the map to me is just so excessive). I'll change it though it is just a better way to do it.